content/docs/tasks/telemetry/export-logs-through-stackdriver.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Exporting Istio Access Logs to BigQuery, GCS, Pub/Sub through Stackdriver
+description: How to export Istio Access Logs to different sinks like BigQuery, GCS, Pub/Sub through Stackdriver.
+weight: 70
+---
+
+This task shows how to export Istio Access Logs to different sinks like
+BigQuery, GCS, Pub/Sub through Stackdriver A user can do analytics on data
```

Comment:
We should fully spell out what `GCS` refers to, and link all the mentioned systems.

Reply:
Done..